ee/modules/650-runtime-audit-engine/docs/ADVANCED_USAGE.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "The runtime-audit-engine module: advanced usage"
+---
+
+## Enabling debugging logs
+
+### Falco
+
+By default, the log level is set to `debug`.
+
+### Falcosidekick
+
+You can enable debugging logs by setting the `.Values.runtimeAuditEngine.debug` value to `true`.
```

Comment:
We need here to add exampe with module config and remove  `.Values.runtimeAuditEngine.debug`

Reply:
done